src/JoomlaBrowser.php

```diff
@@ -439,4 +439,41 @@ public function checkAllResults()
 		$this->debug("Selecting Checkall button");
 		$I->click(['xpath' => "//thead//input[@name='checkall-toggle' or @name='toggle']"]);
 	}
+
+    /**
+     * Function to install a language through the interface
+     *
+     * @param string $languageName Name of the language you want to install
+     *
+     * @return void
+     */
+    public function installLanguage($languageName)
+    {
+        $I = $this;
+        $I->amOnPage('administrator/index.php?option=com_installer&view=languages');
+        $this->debug('I check for Notices and Warnings');
+        $this->checkForPhpNoticesOrWarnings();
+        $this->debug('Refreshing languages');
+        $I->click(['xpath' => "//*[@id=\"toolbar-refresh\"]/button"]);
```

Comment:
after the click I suggest a waitForElement, because the Refresh function could take time, isn't it?